public/components/tweets/tweets.view.html

```diff
@@ -0,0 +1,38 @@
+<nav id="nav">
+    <div class="nav-wrapper">
+        <a class="brand-logo left">
+            Twitter Analyzer
+        </a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+            <li><a ng-click="scrollTo('intro')">About</a></li>
+            <li><a ng-click="scrollTo('analyze')">Analyze</a></li>
+        </ul>
+    </div>
+</nav>
+<div class="heading">
+	<div class="container">
+		<div class="row">
+			<div class="col s12 m10 offset-m1">
+				<h1>@{{twitterHandle}}</h1>
+				<h5>Select the tweet you would like to analyze</h5>
+			</div>
+		</div>
+
+	</div>
+</div>
+<div class="tweets">
+	<div class="container">
+		<div class="row">
+			<div class="col s12 m10 offset-m1">
+			<ul class="collection">
+			<li class="collection-item avatar" ng-repeat="tweet in tweets" ng-click="analyze(tweet)">
+			<img src="../assets/images/bird_chart.jpeg" alt="" class="circle">
+			<span class="title"><b>{{ tweet }}</b></span>
```

Comment:
The `<b>` tag here bothers me.  I would forgo that tag and just add a `font-style: bold;` to the `title` class.  That will eliminate unneeded markup, not to mention that the `<b>` should only be used in rare occasions.